Makefile

```diff
@@ -52,19 +55,20 @@ clean::
 	rm -fr build/
 
 distclean: clean
-	rm -rf $(testdata_objects)
+	rm -rf $(testdata_objects) testdata/firecracker.tgz testdata/firecracker_spec-$(firecracker_version).yaml testdata/LICENSE testdata/NOTICE testdata/THIRD-PARTY
```

Comment:
Mostly a nit but I'm just wondering if there is a way we can further simplify this.

Could we `rm -rfv`a `testdata_dir` ?

Reply:
Good catch. Fixed it.

Reply:
The testdata/ directory has some files. Removing the directory entirely may not work. https://github.com/firecracker-microvm/firecracker-go-sdk/tree/master/testdata

Reply:
I just pushed an up-to-date Makefile, I create a target test_dir which includes all of file that need to be removed. You can find it [HERE](https://github.com/RoyceDavison/firecracker-go-sdk/blob/66460c9184b6ce7df8f03b6f3f477e197b9e4ed5/Makefile#L33). This will not remove the entire files under /testdatda